### PR TITLE
Simplify contributing

### PR DESCRIPTION
**Pull request by zimbatm**
- Highlight the top mistakes directly in CONTRIBUTING.md
- Remove unecessary cruft from the PR template (which I always have to delete anyways)

### CODE REVIEW

**Comment by jagajaga** [comment]
Merge this asap please!

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -12,4 +12,18 @@ under the terms of [COPYING](../COPYING), which is an MIT-like license.
...
+  `(pkg-name | service-name): (from -> to | init at version | refactor | etc)`
+
+  Examples:
+
+  * nginx: init at 2.0.1
+  * firefox: 3.0 -> 3.1.1
+  * hydra service: add bazBaz option
+  * nginx service: refactor config generation
+
+* Don't put dots at the end of package descriptions.
```
Comment (jagajaga) [inline .github/CONTRIBUTING.md]: Also omit pkg's name in the description.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -9,13 +9,5 @@
 - [ ] Tested execution of all binary files (usually in `./result/bin/`)
 - [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).
 
-###### More
-
-Fixes issue #<insert id>
-
-cc @<maintainer>
-
-
```
Comment (grahamc) [inline .github/PULL_REQUEST_TEMPLATE.md]: :+1: Nice to remove these, bonus points on keeping the `---`.

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (zimbatm) [inline .github/CONTRIBUTING.md]: What do you think of this: "meta.description should not contain the package name and should not have a dot at the end".

Comment (joachifm) [inline .github/CONTRIBUTING.md]: May as well throw in that it should be capitalized

Comment (zimbatm) [inline .github/CONTRIBUTING.md]: Like this ? 3b33c3628a6d4f353435002c34183a72e81ff4f6
If it's all good I'll squash and merge.

**Comment by jgillich** [comment]
@aristidb  Please, never ever merge something with `fixup!` in the commit message, rebase it first.

**Pull request merged**